config/routes.rb

```diff
@@ -239,6 +239,7 @@
     resources :exercise_representations
     resources :tracks, only: [:show]
     resources :site_updates, except: [:destroy]
+    resources :teams, except: [:destroy]
```

Comment:
Probably worth naming this `contributor_teams` rather than `teams`, as once we have "real" Exercism teams back in play, this will get confusing. 

_Queue Erik muttering something about C# being better than Ruby 😉_